### PR TITLE
Git, Matrix Configuration Parameter Plugin & Require Upper Bounds redirection placed

### PR DESCRIPTION
**Pull request by krharsh17**
This PR addresses the following issues:
Fixes [#3310](https://github.com/jenkins-infra/jenkins.io/issues/3310)
Fixes [#3311](https://github.com/jenkins-infra/jenkins.io/issues/3311)
Fixes [#3313](https://github.com/jenkins-infra/jenkins.io/issues/3313)

### CODE REVIEW

File: dist/profile/templates/confluence/vhost.conf [hunk 1]
```diff
@@ -2979,6 +2979,8 @@ RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Pipeline\+Nodes\+and\+Processes\+Plugin$" "https://plugins.jenkins.io/workflow-durable-task-step" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Matrix\+Authorization\+Strategy\+Plugin$" "https://plugins.jenkins.io/matrix-auth" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS//Matrix\+Configuration\+Parameter\+Plugin$" "https://plugins.jenkins.io/matrix-combinations-parameter/" [NE,NC,L,QSA,R=301]
```
Comment (MarkEWaite) [inline dist/profile/templates/confluence/vhost.conf]: Need to remove the redundant slash
```suggestion
RewriteRule "^/display/JENKINS/Matrix\+Configuration\+Parameter\+Plugin$" "https://plugins.jenkins.io/matrix-combinations-parameter/" [NE,NC,L,QSA,R=301]
```

File: dist/profile/templates/confluence/vhost.conf [hunk 2]
```diff
@@ -3003,6 +3005,8 @@ RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://jenkins.io/doc/book/m
 RewriteRule "^/display/JENKINS/FreeBSD$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/FreeBSD\+4.9$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+report\+an\+issue$" "https://www.jenkins.io/participate/report-issue/" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS//How\+to\+fix\+RequireUpperBoundDeps$" "https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes" [NE,NC,L,QSA,R=301]
```
Comment (MarkEWaite) [inline dist/profile/templates/confluence/vhost.conf]: Need to remove extra '/' here
```suggestion
RewriteRule "^/display/JENKINS/How\+to\+fix\+RequireUpperBoundDeps$" "https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes" [NE,NC,L,QSA,R=301]
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
The original links in the GitHub issues mistakenly included a duplicated slash inside the URL.  That needs to be removed or the redirect won't work for most users

**Comment by krharsh17** [comment]
Alright, I just applied the changes @MarkEWaite

**Review by MarkEWaite — APPROVED** [review]

**Review by olblak — APPROVED** [review]

**Review by olblak — APPROVED** [review]

**Review by markjacksonfishing — APPROVED** [review]

**Pull request merged**